python/kserve/test/test_storage.py

```diff
@@ -199,8 +199,15 @@ def test_unpack_zip_file():
     os.remove(os.path.join(out_dir, 'model.pth'))
 
 
-def test_storage_spec_update():
-    uri = 's3://<bucket-placeholder>/bar'
```

Comment:
for s3, the path doesn't include bucket name, so we need to have the `<bucket-placeholder>1